packages/headless/src/api/knowledge/stream-answer-api-state.ts

```diff
@@ -0,0 +1,18 @@
+import type {SearchAppState} from '../../state/search-app-state.js';
+import type {
+  ConfigurationSection,
+  GeneratedAnswerSection,
+  GetAnswerQuerySection,
+  InsightConfigurationSection,
+  TabSection,
+} from '../../state/state-sections.js';
+
+export type StreamAnswerAPIState = {
+  searchHub: string;
+  pipeline: string;
```

Comment:
Can you please explain why this is gone from the StreamAnswerAPIState?

`answer: ReturnType<typeof answerApi.reducer>;`

It was in the original `StateNeededByAnswerAPI`